qiita_ware/util.py

```diff
@@ -97,38 +96,6 @@ def per_sample_sequences(iter_, max_seqs, min_seqs=1, random_buf_size=100000):
             yield (sequence_id, sequence)
 
 
-def stats_from_df(df):
```

Comment:
I'm dealing with removals at the end, since tests are failing anyway. That way everything is gutted in one fell swoop.